README.md

```diff
@@ -1,48 +1,101 @@
+
+### Usage
+
+>Clone the app by running this command
+
+```md 
+git clone https://github.com/fobadara/fabulous-vet-clinic
+```
+>Navigate into the directory with
+
+```md
+cd fabulous_vet_clinic
+```
+>To open user
+
+```md
+$  sudo -u postgres psql
+```
+
+>To exit  
+```md
+\q
+```
+or
+
+```md
+exit
+```
+>To navigate into the database
+
+```md
+$  \c vet_clinic
+```
 
+||Built With 🔨 ||
+|-|-------------|-|
+|| ![Postgre](https://img.shields.io/badge/PostgreSQL-316192?style=for-the-badge&logo=postgresql&logoColor=white)
+||
 
-## Authors
 
-👤 **Author1**
+||Tools 🛠️||
+|-|-------------|-|
+||![Git](https://img.shields.io/badge/git-%23F05033.svg?style=for-the-badge&logo=git&logoColor=white)  ![GitHub](https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white)   ![Markdown](https://img.shields.io/badge/markdown-%23000000.svg?style=for-the-badge&logo=markdown&logoColor=white)  ![Visual Studio Code](https://img.shields.io/badge/Visual%20Studio%20Code-0078d7.svg?style=for-the-badge&logo=visual-studio-code&logoColor=white)||
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## Authors ✍️
 
-👤 **Author2**
+| 👤 Abdulfatai Badara  |
+|---|
+|<a target="_blank" href="https://github.com/fobadara"><img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white" alt="Github profile"></a>  <a target="_blank" href="https://www.linkedin.com/in/fob90s"><img src="https://img.shields.io/badge/-LinkedIn-0077b5?style=for-the-badge&logo=LinkedIn&logoColor=white" alt="Linkedin profile"></a> <a target="_blank" href="https://twitter.com/fob90s"><img src="https://img.shields.io/badge/-Twitter-1DA1F2?style=for-the-badge&logo=Twitter&logoColor=white" alt="Twitter profile"></a>  
+<a target="_blank" href="mailto:fob90s@gmail.com"><img src="https://img.shields.io/badge/-Gmail-D14836?style=for-the-badge&logo=Gmail&logoColor=white" alt="Gmail account"></a> <a target="_blank" href="https://wa.me/+2349066478370"> <img src="https://img.shields.io/badge/WhatsApp-25D366?style=for-the-badge&logo=whatsapp&logoColor=white" alt="Whatsapp account"></a> (Click to reach me)|
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the issues page.
 
-## Show your support
+## Show your support 💪
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
 - Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
+This project is licensed by [MIT](LICENSE) -->
```

Comment:
Your project is working **perfectly** but there are some changes that you need to add to make it approvable.
_If you are with me let's get started!!!_ 😉

- [ ] I think you forgot to add LICENCE in your project. Because, when I try to click on it, it gives me a broken link... 
Please avoid this issue first. 😊

Reply:
Thank you. I will make sure I fix this.